src/steps/build-account-to-member-mapping/index.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  Entity,
+  Relationship,
+  createIntegrationRelationship,
+  JobState,
+} from '@jupiterone/integration-sdk';
+import {
+  STEP_ID as MEMBER_STEP,
+  ACCOUNT_MEMBER_TYPE,
+} from '../fetch-enterprise-account-members';
+import {
+  STEP_ID as ACCOUNT_STEP,
+  ACCOUNT_TYPE,
+} from '../fetch-enterprise-accounts';
+
+const step: IntegrationStep = {
+  id: 'build-account-to-member-relationships',
+  name: 'Build Account-to-Member Relationships',
+  types: [],
+  dependsOn: [ACCOUNT_STEP, MEMBER_STEP],
+  async executionHandler({ jobState }: IntegrationStepExecutionContext) {
+    const accountIdMap = await createAccountIdMap(jobState);
+
+    await jobState.iterateEntities(
+      { _type: ACCOUNT_MEMBER_TYPE },
+      async (member) => {
+        const account = accountIdMap.get(member.enterpriseAccountId as number);
+
+        if (account) {
+          await jobState.addRelationships([
+            createAccountToMemberRelationship(account, member),
+          ]);
+        }
+      },
+    );
+  },
+};
+
+export default step;
+
+async function createAccountIdMap(
+  jobState: JobState,
+): Promise<Map<number, Entity>> {
+  const accountIdMap = new Map<number, Entity>();
+  await jobState.iterateEntities({ _type: ACCOUNT_TYPE }, (account) => {
+    // unfortunately need to cast because of EntityPropertyValue type
+    accountIdMap.set(account.id as number, account);
```

Comment:
There is a discussion about making `entity.id` a `string` always: https://lifeomic.slack.com/archives/CK65GRH0F/p1587674334473900. That should be reflected here, right? I suppose you'll want to do as @erkangz did and make `id` something like `{ id: \`heroku-enterprise:${account.id}\`, enterpriseAccountId: account.id}`, and then map in here the entity `enterpriseAccountId` for use later with `team.enterpriseAccountId`.

Reply:
Good catch. `account.id` actually _is_ a string (a uuid) but I borrowed this from Charlie's project and wasn't paying attention to the function's return type.